totest-manager.py

```diff
@@ -373,7 +376,7 @@ def arch(self):
 
     # for Factory we check the version of the release package
     def current_version(self):
-        url = self.api.makeurl(['build', 'openSUSE:%s' % self.project, 'standard', 'x86_64',
+        url = self.api.makeurl(['build', 'openSUSE:%s' % self.project, 'standard', self.arch(),
```

Comment:
I can't see the point for this line change, ToTestFactory.arch() is always return x86_64 no? as ToTestFactoryPowerPC.arch() always return ppc64le.